lib/ansible/modules/cloud/amazon/route53_zone.py

```diff
@@ -323,24 +378,22 @@ def main():
 
     private_zone = bool(vpc_id and vpc_region)
 
-    _, _, aws_connect_kwargs = get_aws_connection_info(module)
-
     # connect to the route53 endpoint
-    try:
-        conn = Route53Connection(**aws_connect_kwargs)
-    except boto.exception.BotoServerError as e:
-        module.fail_json(msg=e.error_message)
+    region, ec2_url, aws_connect_kwargs = get_aws_connection_info(module, boto3=True)
```

Comment:
Instead of all this, `client = module.client('route53')` will now get a properly configured Route53 client (or `module.resource('route53')` for the resource-style SDK if you need that).

Reply:
Indeed, much simpler :)